admin_guide/scheduling/pod_placement.adoc

```diff
@@ -293,26 +293,29 @@ endif::[]
 . Create a project object that includes the
 `scheduler.alpha.kubernetes.io/node-selector` annotation and labels.
 +
+[source,yaml]
 ----
-{
-    "kind": "Namespace",
-    "apiVersion": "v1",
-    "metadata": {
-        "name": "ns1",
-        "annotations": {
-            "scheduler.alpha.kubernetes.io/node-selector": "env=test,infra=fedora" <1>
-        }
-    },
-    "spec": {},
-    "status": {}
-}
-
+apiVersion: v1
+kind: Namespace
+metadata
+  name: ns1
+  annotations: 
+    scheduler.alpha.kubernetes.io/node-selector: env=test,infra=fedora <1>
+spec: {},
+status: {}
 ----
 +
 <1> Annotation to create the labels to match the project label selector. Here, the key/value labels are `env=test` and `infra=fedora`.
++
+[NOTE]
+====
+When using the *Pod Node Selector* admission controller, you cannot use `oc adm new-project <project-name>` for setting project node selector. 
+The `oc adm new-project` command uses the `NodeEnv` admission controller.
```

Comment:
Sorry to be nit-picking here. I'd recommend something like (words you can change whatever way you want): when setting project node selectors via `oc adm new-project myproject --node-selector='type=user-node,region=east`, it sets following annotation: `openshift.io/node-selector` and this annotation is processed/operated/used by NodeEnv admission plugin. Just trying to be as clear as possible.